lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -669,7 +669,8 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
 
         if not subscription:
             raise LisaException(
-                f"cannot find subscription id: '{self.subscription_id}'"
+                f"cannot find subscription id: '{self.subscription_id}', "
+                f"make sure exists and current account can access"
```

Comment:
How about we start phrases with uppercase letters? Also, "Make sure it exists and current account can access it".